airflow-core/src/airflow/models/trigger.py

```diff
@@ -98,6 +98,7 @@ class Trigger(Base):
     encrypted_kwargs: Mapped[str] = mapped_column("kwargs", Text, nullable=False)
     created_date: Mapped[datetime.datetime] = mapped_column(UtcDateTime, nullable=False)
     triggerer_id: Mapped[int | None] = mapped_column(Integer, nullable=True)
+    queue: Mapped[str] = mapped_column(String(128), nullable=True)
```

Comment:
in airflow-core/src/airflow/models/taskinstance.py:404 the queue field is defined to be 256, so should be the same in the triggerer
```suggestion
    queue: Mapped[str] = mapped_column(String(256), nullable=True)
```

Reply:
Updated, and added a unit test in `test_trigger.py:: test_queue_column_max_len_matches_ti_column_max_len ` to prevent this from happening in the future.